client/components/CreateEventForm/CreateEventFormDumb.js

```diff
@@ -39,31 +40,39 @@ class CreateEventForm extends Component {
       };
 
     handleSubmit = async e => {
-    e.preventDefault();
+        e.preventDefault();
 
-    try {
-        const { data } = await Axios.put('/auth/local/login', this.state.values, {
-        validateStatus: function(status) {
-            return status === 200 || status === 401;
-        },
-        });
+        try {
+            const { data } = await Axios.post('/event/newevent', 
+                this.state.values,
```

Comment:
Thanks for making this post @katherinepeterson. I was wondering why this was a put request...